Lesson2/src/main/java/pl/solvd/university/people/Applicant.java

```diff
@@ -65,7 +65,7 @@ public void setActivity(Activity activity) {
     }
 
     public String getFirstLastName() {
-        return String.format("%s %s", this.firstName, this.lastName);
+        return firstLastName = String.format("%s %s", this.firstName, this.lastName);
```

Comment:
why do you need firstLastName here?

Reply:
do not get the question. What for a variable firstLastName in global sense? why is it needed at all?